guides/scripts/satellite-sync.sh

```diff
@@ -90,12 +112,12 @@ if ls $US_REPO/guides/doc-* 1> /dev/null 2>&1; then
   cp -r $US_REPO/guides/doc-Administering_Red_Hat_Satellite/* $DS_REPO/doc-Administering_Red_Hat_Satellite/
   cp -r $US_REPO/guides/doc-Configuring_Load_Balancer/* $DS_REPO/doc-Load_Balancing_Guide/
   cp -r $US_REPO/guides/doc-Planning_Guide/* $DS_REPO/doc-Architecture_Guide/
-
+  cp $US_REPO/guides/scripts/satellite-sync.sh $DS_REPO/scripts/satellite-sync.sh
 fi
 # Add downstream build attributes and find and replace attributes with fixed terms
 
 cd $DS_REPO
-sed 's/:SatelliteAnsibleVersion: 2.8/:SatelliteAnsibleVersion: 2.8\n:build: satellite/g' common/attributes.adoc > common/attributes.tmp
+sed '1s/^/\:build\: satellite\n/' common/attributes.adoc > common/attributes.tmp
 mv common/attributes.tmp common/attributes.adoc
 sed 's/:TargetVersion: 6.7-beta/:TargetVersion: 6.7/g' common/attributes.adoc > common/attributes.tmp
 mv common/attributes.tmp common/attributes.adoc
```

Comment:
@melcorr What does lines 122-127 in this file do? Please elaborate why they replace version numbers. I would probably add comments about it to make this clear for future generations.

Note the change that I've done on line 120. The attribute `SatelliteAnsibleVersion` will change with time, so instead of using it as a reference point, I made `sed` to add the line `:build\: satellite` in the beginning of attributes.adoc and put a newline after it.

Reply:
Sergei, the link checker fails and all the docs links are broken if we use the latest release number in the upstream. 
This was the workaround we came up with. 
I have absolutely no idea what you mean by build there. Sorry. Makes no sense to me.

Reply:
> Sergei, the link checker fails and all the docs links are broken if we use the latest release number in the upstream.
> This was the workaround we came up with.
Got it. For this release, I solved it by keeping the working `6.7` version on the upstream master branch, but changing the version to `6.8-beta` on upstream SATELLITE-6.8-beta and downstream 6.8-beta branches. I will probably remove this workaround to avoid confusion.
> I have absolutely no idea what you mean by build there. Sorry. Makes no sense to me.
Anyway, this works :) So the `sed` command on line 129 inserts the line `:build: satellite` after the line `:SatelliteAnsibleVersion: 2.8` in the downstream attributes.adoc. I changed it to insert `:build: satellite` at the beginning of the attributes.adoc file.

Reply:
All of this was experimental at the time. If it works for you, it works :)